bootstrap/src/main/kotlin/jp/co/soramitsu/bootstrap/genesis/d3/D3TestGenesisFactory.kt

```diff
@@ -14,7 +14,15 @@ import java.util.*
 class D3TestGenesisFactory : GenesisInterface {
 
     private val zeroPubKey = "0000000000000000000000000000000000000000000000000000000000000000"
-    override fun getAccountsNeeded(): List<AccountPrototype> = D3TestContext.d3neededAccounts
+    override fun getAccountsForConfiguration(peersCount: Int): List<AccountPrototype> {
+        val activeAccounts = D3TestContext.d3neededAccounts.filter { !it.passive }
+        activeAccounts.stream().forEach {
```

Comment:
Can we remove `stream()`? Is it going to work without this call?

Reply:
fixed